README.md

```diff
@@ -515,6 +517,16 @@ A `service_discovery` block configures [Amazon Service Discovery](https://docs.a
 
 * `namespace_id` – (Required) The ID of the namespace to use for DNS configuration.
 
+* `container_name` - (Required) The name of the container to use
```

Comment:
The service discovery map does not support container_name as input.

Reply:
I will remove this line from the file.